guides/release/object-model/enumerables.md

```diff
@@ -4,6 +4,11 @@ objects, and which allows you to work with those children using the
 enumerable in the majority of apps is the native JavaScript array, which
 Ember.js extends to conform to the enumerable interface.
 
+*Note:* Ember used to enabled prototype extensions by default. This can get messy,
```

Comment:
```
If you are using [Ember Fastboot](https://ember-fastboot.com/), we recommend 
[disabling prototype extensions](https://guides.emberjs.com/release/configuring-ember/disabling-prototype-extensions/) 
and always manually wrapping your arrays and objects when you need these methods.
```

I think this should give enough info for an advanced Ember dev to know what to do, while avoiding confusing a newer Ember dev.

Reply:
Attempt # 2:

```
It is best practice to [disable prototype extensions](https://guides.emberjs.com/release/configuring-ember/disabling-prototype-extensions/) in your app. For example, this means that Ember's convenience methods like `firstObject` will only be available on `EmberArray`s and will not work on Native JavaScript arrays. This is especially important for apps that use [Ember Fastboot](https://ember-fastboot.com/).
```